### PR TITLE
✨ Add task queue info endpoint

### PR DESCRIPTION
**Pull request by phlmn**
Needed for autoscaling

### CODE REVIEW

File: backend/transcribee_backend/models/task.py [hunk 1]
```diff
@@ -64,6 +65,24 @@ class Task(TaskBase, table=True):
     state: TaskState = TaskState.NEW
     state_changed_at: datetime.datetime = Field(default_factory=now_tz_aware)
 
+    initial_cost: (float | None) = None
+
+    def guess_initial_cost(self, media_duration_sec: float) -> float | None:
```
Comment (pajowu) [inline backend/transcribee_backend/models/task.py:70]: Why not calculate this as-needed and make this a property function?
```suggestion
    @property
    def initial_cost(self, media_duration_sec: float) -> float | None:
```

File: backend/transcribee_backend/routers/document.py [hunk 2]
```diff
@@ -521,6 +521,11 @@ def set_duration(
 ) -> ApiDocument:
     doc = task.document
     doc.duration = body.duration
+
+    # update task costs based on duration
+    for task in doc.tasks:
+        task.initial_cost = task.guess_initial_cost(doc.duration)
+
```
Comment (pajowu) [inline backend/transcribee_backend/routers/document.py:528]: Can be removed if initial_cost is a property

File: backend/transcribee_backend/models/task.py (hunk 1 above)
Comment (phlmn) [reply backend/transcribee_backend/models/task.py:70]: We can do that, but then we have to query the document each time

Comment (phlmn) [reply backend/transcribee_backend/models/task.py]: done

**Pull request merged**